src/lib/users/dialogs/inst_user_first.rb

```diff
@@ -538,9 +539,12 @@ def import_available?
     #
     # @see Y2Users::InstUsersDialogHelper#user_to_validate
     #
-    # @return [Y2Users::User] root user if using entered password for it too; the new user otherwise
+    # @return [Y2Users::User] a root user when using same password for root; a normal user otherwise
     def user_to_validate
-      @use_pw_for_root ? root_user : user
+      username = @user_pw_for_root ? "root" : "user"
```

Comment:
I would avoid to create a temporary user only for validations. This could lead to some troubles if we do not create the user rightly. I think the dialog can directly work with a user that is not attached to any config and then it can perform validations over such a user. The resulting user from the dialog could be attached/updated in the corresponding config once the dialog is accepted. And, if the dialog is discarded, then the user is simply gone.

Reply:
As we have discussed, this can be addressed in a separate PR.